funcx_endpoint/funcx_endpoint/endpoint/register_endpoint.py

```diff
@@ -75,3 +82,41 @@ def register_endpoint(
         )
 
     return reg_info["task_queue_info"], reg_info["result_queue_info"]
+
+
+def _get_endpoint_metadata(config: Config) -> dict:
+    metadata: t.Dict[str, t.Any] = dict()
```

Comment:
This is the default dict type, I believe, so either:
```python
...
def _get():
    metadata = {}
    ...
```
or
```python
from __future__ import annotations
...
def _get():
    metadata: dict = {}
    metadata: dict[str, t.Any] = {}  # alternatively
```

Reply:
The `metadata = {}` line doesn't pass `mypy` on my machine, but the `metadata: dict = {}` does and is cleaner; updated.